### PR TITLE
Add da emobil (operator for charging_station)

### PR DESCRIPTION
**Pull request by mcliquid**
Currently more than 500 stations.
See https://www.da-emobil.com/

### CODE REVIEW

File: data/operators/amenity/charging_station.json [hunk 1]
```diff
@@ -788,6 +788,14 @@
         "operator": "Daimler AG"
       }
     },
+    {
+      "displayName": "da emobil GmbH & Co KG",
```
Comment (bmillemathias) [inline data/operators/amenity/charging_station.json:792]: There should be a capital letter of firast letter of words so "Da Emobil ..."

File: data/operators/amenity/charging_station.json [hunk 2]
```diff
@@ -788,6 +788,14 @@
...
+      "locationSet": {"include": ["at", "de", "ch", "gb", "ie", "it", "fr", "hu", "dk"]},
+      "tags": {
+        "amenity": "charging_station",
+        "operator": "da emobil GmbH & Co KG"
```
Comment (bmillemathias) [inline data/operators/amenity/charging_station.json:796]: same here

**Review by bmillemathias — CHANGES_REQUESTED** [review]

File: data/operators/amenity/charging_station.json (hunk 1 above)
Comment (bhousel) [reply data/operators/amenity/charging_station.json:792]: Here the display name should just be "da emobil".  This is the name that you'd see in the JOSM menu, or in the iD preset, not the legal name.

**Comment by bmillemathias** [comment]
if itś done on purpose so I think that's ok

File: data/operators/amenity/charging_station.json (hunk 2 above)
Comment (bhousel) [reply data/operators/amenity/charging_station.json:796]: Here you can use whatever `operator` value the local community prefers.  Most local communities don't like to see the "GmBH & Co KG" stuff, but maybe yours does?

The important thing for this to be converted to a preset is to add a `operator:wikidata` tag  (which this PR doesn't have).

**Comment by LaoshuBaby** [comment]
> if itś done on purpose so I think that's ok

Another thing is data in `tags` will be added to OSM element, so they must fit `on the ground` rule, if it was called `da emobil` more offen, you can just it as operator.

For example, in China we only need a `operator=星巴克` for starbucks, not need `operator=星巴克企业管理（中国）有限公司`, that is "Starbucks ~~Enterprise Management (China) Co., Ltd.~~ "

Many country's company will have special surfix, like "Kabushiki gaisha"(株式会社) in Japan, but you haven't see local community add it as operator because its official name, not most common used name. The uppercase problem is also like this, if you seldom see a uppercased name, just leave it as is.

Of course, I don't live in Germany, so use which name should be chosen by local community.

**Review by arch0345 — APPROVED** [review]
Thanks @mcliquid!

**Pull request merged**